optimisation/index.md

```diff
@@ -1,7 +1,6 @@
 # Module content
 
-- ToDo
-- ToDo
-- ToDo
-- ToDo
-- ToDo
+1. Intro (how does RN works, why performance issues can be faced)
+2. How to measure performance of RN app
+3. Common ways to improve performance (console statements; FlatList; React.memo(); useMemo()/useCallback(); images: sizes and format, caching; aminations; unnecessary dependencies; Reselect; inline anonymous functions
+4. Hermes 
```

Comment:
What do you think about adding Analytics section here? (Crashlytics/Firebase/Sentry)?

Reply:
will cover this in Analytics module #58